src/Bicep.LangServer/Completions/PublicRegistryModuleMetadataProvider.cs

```diff
@@ -29,44 +31,43 @@ namespace Bicep.LanguageServer.Providers
     /// </summary>
     public class PublicRegistryModuleMetadataProvider : IPublicRegistryModuleMetadataProvider
     {
-        private record ModuleTagPropertiesEntry(string description);
-        private record ModuleMetadata(string moduleName, List<string> tags, Dictionary<string /*key: tag*/, ModuleTagPropertiesEntry>? properties);
+        private record ModuleTagPropertiesEntry(string Description, string DocumentationUri);
+
+        private record ModuleMetadata(
+            string ModuleName,
+            List<string> Tags,
+            ImmutableDictionary<string /*key: tag*/, ModuleTagPropertiesEntry> Properties);
+
+        private const string LiveDataEndpoint = "https://aka.ms/br-module-index-data";
+
+        private static readonly JsonSerializerOptions JsonSerializerOptions = new()
+        {
+            PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+        };
+
+        private readonly HttpClient httpClient;
 
-        private const string LiveDataEndpoint = "https://aka.ms/BicepModulesMetadata";
         private readonly TimeSpan CacheValidFor = TimeSpan.FromHours(1);
```

Comment:
I don't want you to bother changing this back, but why add all the whitespace?  This style doesn't improve readability, while grouping by similarity IMHO does.

Reply:
It's a convention that I'm used to (not sure where I learned it from, maybe ReSharper or StyleCop), and I find it slightly more readable with line breaks between fields. I'm have no objections to removing the line breaks, but personally I'm not a fan of grouping fields by similarity, and I'd follow https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1201.md for sorting class members.